features/step_definitions/hutSteps.js

```diff
@@ -11,21 +11,21 @@ When('I register a hut with the following information:', async function (dataTab
         zipCode,
         email,
         phone,
-        prices
+        dayPrices
     } = dataTable.hashes()[0];
 
-    let pricesCleaned = prices.split(',').map(p => parseInt(p.trim()));
-    let pricesObject = {
-        monday: pricesCleaned[0],
-        tuesday: pricesCleaned[1],
-        wednesday: pricesCleaned[2],
-        thursday: pricesCleaned[3],
-        friday: pricesCleaned[4],
-        saturday: pricesCleaned[5],
-        sunday: pricesCleaned[6]
+    let dayPricesCleaned = dayPrices.split(',').map(p => parseInt(p.trim()));
```

Comment:
You can get the variables like this, which provides a bit of context for why you are splitting the string:

```js
let [ monday, tuesday, wednesday, thursday, friday, saturday, sunday ] = dayPrices.split(',').map(p => parseInt(p.trim()));
```

Use it if you think it's better -- it's up to you :smile:

Reply:
much nicer :D